zonemaster/zonemaster-gui
# Update Danish translation for tool tips

## Purpose

PR #243 adds mechanism for translation of tool tips. 

## Context

Resolves #244 with data provided by @haarbo in #244.

## Changes

Adds Danish translation to the tool tips in da.json.

## How to test this PR

Open Zonemaster GUI with this PR included, test some domain, switch to Danish translation and move the mouse over the "Info", "Warning" etch buttons and verify that the tool tips is in Danish.
